libs/core/src/lib/list/list-navigation-item/list-navigation-item.component.ts

```diff
@@ -70,13 +97,53 @@ export class ListNavigationItemComponent implements AfterContentInit {
         this._innerText = this._text.elementRef.nativeElement.innerText;
     }
 
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this._setIsItemVisible(this.expanded);
+    }
+
     /** @hidden */
     @HostListener('click', ['$event'])
     onItemClick(event: MouseEvent): void {
         event.stopPropagation();
         this._handleExpandedChanges(!this.expanded);
     }
 
+    /** @hidden */
+    @HostListener('keydown', ['$event'])
+    keyDownHandler(event: KeyboardEvent): void {
+        if (KeyUtil.isKeyCode(event, [RIGHT_ARROW])) {
```

Comment:
Really nice work! Only thing I would change is I think these left/right arrow checks need to check for RTL mode, I think a left arrow press should expand and a right arrow should collapse when in RTL mode. Other than that I think ready to merge

Reply:
Hm, agree. It should also be enabled for [Side Navigation](https://fundamental-ngx.netlify.app/#/core/side-navigation#programmatic-state-change).